src/services/Store/store/asset.slice.ts

```diff
@@ -78,7 +78,14 @@ export default slice;
  * Selectors
  */
 
-export const getAssets = createSelector([getAppState], (s) => s[slice.name]);
+export const getAssets = createSelector([getAppState], (s) => s.assets);
+export const getSwapAssets = createSelector([getAssets], (assets) =>
+  assets.filter((a) => a.isCustom || a.isSwapRelevant)
```

Comment:
are custom assets always swap-supported? would probably prefer to exclude unless we know for certain

Reply:
The 0x API supports all ERC20s, it will simply fail if there isn't enough liquidity. This is why we landed on using the filtered list + custom tokens.